### DISCUSSION TITLE
Please add directory navigation bar for the README.md page

### DISCUSSION THREAD

**Discussion by songyuc** [community/community#48636] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi, guys!
I hope to add a directory navigation bar for the README.md page, which will be handier for me to get an outline for the current README documentation.
For example, as for [ML-visuals README](README),
![图片](https://user-images.githubusercontent.com/27288110/221334595-37e537ce-4027-44de-85d6-b5caafc5c33c.png)
there will be a directory navigation bar, like:
1. ML Visuals
2. How to Use?
3. Versions:
4. ...

Then I can get the main outline.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @songyuc 

There is this option in the upper left corner, which has the topics.

![image](https://user-images.githubusercontent.com/47532703/221423001-f3f2e6f0-bf6a-4089-9957-8838c841c655.png)

**Reply by songyuc** [reply]
Thanks sincerely for your guide!
